x/valset-pref/keeper.go

```diff
@@ -37,21 +38,76 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", fmt.Sprintf("x/%s", types.ModuleName))
 }
 
+// SetValidatorSetPreferences sets a new valset position for a delegator in modules state.
 func (k Keeper) SetValidatorSetPreferences(ctx sdk.Context, delegator string, validators types.ValidatorSetPreferences) {
 	store := ctx.KVStore(k.storeKey)
 	osmoutils.MustSet(store, []byte(delegator), &validators)
 }
 
+// GetValidatorSetPreference returns the existing valset position for a delegator.
 func (k Keeper) GetValidatorSetPreference(ctx sdk.Context, delegator string) (types.ValidatorSetPreferences, bool) {
 	store := ctx.KVStore(k.storeKey)
 	bz := store.Get([]byte(delegator))
 	if bz == nil {
 		return types.ValidatorSetPreferences{}, false
 	}
+
+	// valset delegation exists, so return it
 	var valsetPref types.ValidatorSetPreferences
 	if err := proto.Unmarshal(bz, &valsetPref); err != nil {
 		return types.ValidatorSetPreferences{}, false
 	}
 
 	return valsetPref, true
 }
+
+// GetDelegations checks if valset position exists, if it does return that
+// else return existing delegation that's not valset.
+func (k Keeper) GetDelegations(ctx sdk.Context, delegator string) (types.ValidatorSetPreferences, error) {
```

Comment:
Consider changing this to `GetDelegationPreferences`, a part of this methods's functionality is getting val set preference if it exists, I feel like the term of getting delegation can cause confusion

```suggestion
func (k Keeper) GetDelegationPreferences(ctx sdk.Context, delegator string) (types.ValidatorSetPreferences, error) {
```